### PR TITLE
Make sure the benchmark build config is tested on trunk for easy bisect

### PR DESCRIPTION
**Pull request by ezyang**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #142376

Signed-off-by: Edward Z. Yang <ezyang@meta.com>

### CODE REVIEW

**Review by atalman — APPROVED** [review]
lgtm

File: .github/workflows/inductor-perf-test-nightly.yml [hunk 1]
```diff
@@ -76,6 +76,7 @@ jobs:
       curr_branch: ${{ github.head_ref || github.ref_name }}
       curr_ref_type: ${{ github.ref_type }}
 
+  # NB: Keep this in sync with trunk.yml
```
Comment (malfet) [inline .github/workflows/inductor-perf-test-nightly.yml:79]: @suo added `sync-tag:` and linter in https://github.com/pytorch/pytorch/blob/main/tools/linter/adapters/workflow_consistency_linter.py a while back, would be nice to use here

File: .github/workflows/trunk.yml [hunk 2]
```diff
@@ -264,3 +264,14 @@ jobs:
       docker-image: ${{ needs.linux-focal-cuda12_4-py3_10-gcc9-experimental-split-build.outputs.docker-image }}
       test-matrix: ${{ needs.linux-focal-cuda12_4-py3_10-gcc9-experimental-split-build.outputs.test-matrix }}
     secrets: inherit
+
+  # NB: Keep this in sync with inductor-perf-test-nightly.yml
+  linux-focal-cuda12_1-py3_10-gcc9-inductor-build:
```
Comment (malfet) [inline .github/workflows/trunk.yml:269]: CUDA-12.1 again... Noooo
@atalman weren't migration over?

Comment (atalman) [reply .github/workflows/trunk.yml:269]: @malfet  sorry missed this. Yes this should be 12.4 you totally right

Comment (ezyang) [reply .github/workflows/trunk.yml:269]: The performance benchmarks are still on 12.4. So you need to migrate the perf benchmarks...

**Pull request closed without merging**